core/src/calleeFunctions/helpers/oneInch.ts

```diff
@@ -94,7 +94,7 @@ export async function getOneinchSwapParameters(
     collateralSymbol: string,
     amount: string,
     marketId: string,
-    slippage = '10'
+    slippage = '1'
```

Comment:
Why was it set to 10 initially?

Reply:
there was no evidence that slippage `1` does not work out and has to be replaced. It's probably just a typo, which i noticed and adjusted to the original script's value.